action-a/entrypoint.sh

```diff
@@ -0,0 +1 @@
+sh -c "echo Hello world my name is $MY_NAME"
```

Comment:
@mattkrick this should fix it!

```suggestion
#!/bin/sh -l

sh -c "echo Hello world my name is $MY_NAME"
```

Reply:
awesome! thank you!